crates/criticalup-cli/src/cli/subcommand/init.rs

```diff
@@ -0,0 +1,45 @@
+// SPDX-FileCopyrightText: The Ferrocene Developers
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+use crate::cli::CommandExecute;
+use crate::errors::Error;
+use crate::Context;
+use clap::Parser;
+use criticalup_core::project_manifest::v1::sample_manifest;
+use tokio::fs::File;
+use tokio::io::{AsyncWriteExt, BufWriter};
+
+/// Create a manifest file (criticalup.toml) inside current directory
+#[derive(Debug, Parser)]
+pub(crate) struct Init {
+    /// Release version string of Ferrocene for this manifest
+    #[arg(long)]
+    release: String,
+    /// Only print the contents of manifest instead of saving to file
+    #[arg(long)]
+    print_only: bool,
```

Comment:
Do you think `--print` would be sufficient for this flag without muddling meaning?

Reply:
I swear I saw in one of the upstream tools about `print-only` but now I cannot find it. I made it `--print`.

Reply:
The closest I could think of was `cargo init` and it has no similar flag.